gbus/tx/mysql/txoutbox.go

```diff
@@ -38,20 +38,24 @@ type TxOutbox struct {
 	exit chan bool
 }
 
+func (outbox *TxOutbox) log() *log.Entry {
+	return log.WithField("_service", outbox.svcName)
```

Comment:
Why _service and not just service?

Reply:
so that it would be the first field displayed by logurs...helps in our testing